package.js

```diff
@@ -6,11 +6,11 @@ Package.describe({
 });
 
 var npmModules = {
-  "debug": "0.7.4",
+  "debug": "4.1.1",
```

Comment:
New versions of debug use const/let, which breaks using in Meteor 1.4 and 1.5.
https://travis-ci.org/monti-apm/monti-apm-agent/jobs/557454465
https://travis-ci.org/monti-apm/monti-apm-agent/jobs/557454466

Reply:
I see. I forgot Meteor 1.4 and 1.5 doesn't support ES6. I think it's the same for Lru-cache.

Reply:
So, I switch to debug 0.8.1 (Apr 15, 2014) and lru-cache 4.1.5.